src/Inventory/Request.php

```diff
@@ -349,6 +349,12 @@ public function inventory($data)
         /** @var array $CFG_GLPI */
         global $CFG_GLPI;
 
+        $data = Plugin::doHook(Hooks::PRE_INVENTORY, $data);
+        if ($data === null) {
+            $this->addError('Rejected by a plugin', 400);
+            return;
+        }
```

Comment:
Why not eventually handle here an error reported by the plugin so the agent can log the real reason ?

Reply:
done

Reply:
Thank you